packages/ui-avatar/src/Avatar/index.tsx

```diff
@@ -122,13 +127,18 @@ class Avatar extends Component<Props> {
   state = { loaded: false }
 
   componentDidMount() {
-    // @ts-expect-error ts-migrate(2722) FIXME: Cannot invoke an object which is possibly 'undefin... Remove this comment to see the full error message
     this.props.makeStyles(this.state)
   }
 
-  componentDidUpdate() {
-    // @ts-expect-error ts-migrate(2722) FIXME: Cannot invoke an object which is possibly 'undefin... Remove this comment to see the full error message
-    this.props.makeStyles(this.state)
+  componentDidUpdate(prevProps: Props, prevState: State) {
+    let updateConfig: MakeStyleConfig | undefined
+    if (prevState.loaded == this.state.loaded) {
+      updateConfig = {
+        prevProps: prevProps,
+        styledProps: ['size', 'shape', 'src']
+      }
+    }
+    this.props.makeStyles(this.state, updateConfig)
```

Comment:
This would be the call site for the 1 parameter `makeStyles`:
```js
componentDidUpdate(prevProps: Props, prevState: State) {
    let updateConfig
    if (prevState.loaded == this.state.loaded) {
      updateConfig = {
        prevProps: prevProps,
        styledProps: ['size', 'shape', 'src']
      }
    }
    this.props.makeStyles({ updateConfig, ...this.state })
  }

```